lib/OpenQA/Schema/ResultSet/JobSettings.pm

```diff
@@ -6,6 +6,11 @@ use Mojo::Base 'DBIx::Class::ResultSet', -signatures;
 
 use OpenQA::App;
 
+sub all_values_sorted ($self, $job_id, $key) {
+    state $options = {distinct => 1, columns => 'value', order_by => 'value'};
```

Comment:
Why are you using `state` here?

Reply:
Because this hash never changes, so no need to re-initialize it on every function call.

Reply:
It's a bit uncommon for a simple, small hashref, but I won't complain